### ISSUE TITLE
您好，关于训练结果可视化的问题。

### ISSUE THREAD

**Issue by DLMU-dh218** [lehaifeng/T-GCN#45]
您好，打扰您一下，我用的是pytorch版本的代码，第55行代码也修改成为loss = self.loss(predictions, y)，在电脑上运行完毕后，正确率一直是在85%左右。
第二个问题是，最终的训练结果可视化，在代码中，是定义了显示图片的函数，但是运行结束后，似乎是没有图片显示出来的
[2021-04-12 09:57:37,792 INFO]val_loss│RMSE│MAE│accuracy│R2│ExplainedVar│train_loss
─────────────────────────────────────────────────────
73.09178161621094│8.549372673034668│6.055265426635742│0.8544564843177795│0.6201916933059692│0.6203931570053101│0.007503977511078119

**Comment by martinwhl** [comment]
Hi there,

1. Could you please provide your hyperparameter settings?
2. Run Tensorboard to show the prediction results, for example:
```bash
tensorboard --logdir lightning_logs/version_0
```
For more help, you can refer to the documents of PyTorch Lightning.

**Comment by DLMU-dh218** [comment]
thank you for your reply.
i didn’t change any hyperparameter.i run the code  in its default settings and environment.

**Comment by martinwhl** [comment]
Please follow the hyperparameter settings in README.
